README.md

```diff
@@ -29,6 +29,14 @@ Drag mouse around the screen to create fluid particles with velocities in
 direction of mouse travel. Fluid fades slowly over time so as not to fill
 the container.
 
+### Parallel Inclusive Scan
+*** This example is work in progress so may not work *** \
+Implementation of a parallel inclusive scan with a given associative binary operation in SYCL.
+
+### Matrix Multiply OpenMP Comparison
+*** This example is work in progress so may not work *** \
+A block tiled matrix multiplication example which compares an OpenMP blocked matrix multiplication implementation with a SYCL blocked matrix multiplication example. The purpose is not to compare performance, but to show the similarities and differences between them. See block_host for the OpenMP implementation.
```

Comment:
It would be great if we could stick to the same 80-column limit as the rest of this file.

Reply:
Is this a comment or a request to make changes? I personally do not mind text lines beyond 80 columns. Why 80 columns.  We all use monitors with high resolution very capable to display long lines of text. An asciidot /.md is purposefully designed to have free following text to automatically fit the media or viewport it is to be displayed in. It cannot be limited to 80 characters width. There is not a setting for it to do this.

Reply:
Changes made.
Done.